0001-basic-expiry-model.md

```diff
@@ -0,0 +1,94 @@
+# Basic model to calculate expiry times
+
+* Status: proposed
+* Authors: [Lloyd Fournier](@LLFourn), [Lucas Soriano](@luckysori)
+* Date: 2019-03-05
+
+Tracking issue: [#14](https://github.com/comit-network/RFCs/issues/14)
+
+## Context
+
+The protocol defined in [RFC003](https://github.com/comit-network/RFCs/blob/master/RFC-003-SWAP-basic.md) introduces the concept of expiry times, which mark the time when assets can be refunded to their original owners. Values that are reasonable and that ensure that the protocol is secure for both parties must be estimated.
+
+## Research
+
+### Findings
+
+@LLFourn and I had a couple of meetings and we figured out some things:
+
+- The difference between `alpha_expiry` and `beta_expiry` has an effect on the security of the model for *Bob*. This does not affect Alice since she will have already redeemed `beta_asset` by the time this is relevant.
+- `beta_expiry` has an effect on the time that *Alice* will have to redeem `beta_asset` and therefore on the likelihood that the protocol will be completed successfully. This does _not_ have any direct effect on the security of the protocol, but rather on its *execution guarantees*.
```

Comment:
```suggestion
- `beta_expiry` has an effect on the time that *Alice* will have to redeem `beta_asset` and therefore on the likelihood that the protocol will be completed successfully. This does _not_ have any direct effect on the security of the protocol, but rather on its *execution guarantee*.
```

Reply:
the guarantee of its execution. Not sure why this is plural.

Reply:
We use both ways in the original [OKR issue](https://github.com/comit-network/comit-rs/issues/670), but I prefer "execution guarantees". I asked @LLFourn to write the next draft anyway, so he can choose or just remove it.